### PR TITLE
EPMLABSBRN-637_Add_logo

### PR DESCRIPTION
**Pull request by Mikleprog**
[EPMLABSBRN-637](https://jira.epam.com/jira/browse/EPMLABSBRN-637)

Adeed logo JetBrain

### CODE REVIEW

File: frontend/app/components/footer/template.hbs [hunk 1]
```diff
@@ -4,6 +4,11 @@
       {{t "footer.info_about"}}
     </span>
     <ul class="flex items-center">
+      <li data-test-support-logo>
+        <a href="https://www.jetbrains.com/" target="blank" rel="noopener">
+          <img src="/logos/jetbrains-variant-4.png" alt="JetBrains" />
```
Comment (lifeart) [inline frontend/app/components/footer/template.hbs:9]: let's optimize png using https://squoosh.app/
to reduce download size

Comment (Mikleprog) [reply frontend/app/components/footer/template.hbs]: Done,  I have to use another source to reduce size

Comment (lifeart) [reply frontend/app/components/footer/template.hbs]: I see 256kb file size, it’s too big for logo

Comment (lifeart) [reply frontend/app/components/footer/template.hbs]: image
it's possible to reduce image size to 23.8kb, plz follow it

Comment (Mikleprog) [reply frontend/app/components/footer/template.hbs]: Fixed

**Review by lifeart — APPROVED** [review]

**Pull request merged**